python_components/classifier/classifier.py

```diff
@@ -163,9 +161,14 @@ def get_predictions(feature_matrix, model_path):
     )
     args = parser.parse_args()
 
+    labels = get_labels()
+    label_mapping = {ind: label for label, ind in labels.items()}
+
     pdf_features = get_features(args.pdfs_path)
     pdf_feature_matrix = get_feature_matrix(pdf_features)
+    del pdf_features
     predictions, confidences = get_predictions(pdf_feature_matrix, "xgboost_model.json")
+    del pdf_feature_matrix
```

Comment:
Deleting these objects just saves memory when running this script on really large CSVs. Wasn't something that came up for Georgia but was needed for Austin.